org-code-javabuilder/lib/src/main/java/org/code/neighborhood/Painter.java

```diff
@@ -0,0 +1,144 @@
+package org.code.neighborhood;
+
+public class Painter {
+    private int xLocation;
+    private int yLocation;
+    private Direction direction;
+    private int remainingPaint;
+    private Grid grid;
+    private String id;
+
+    public Painter(int x, int y, String direction, int paint) {
+        this.xLocation = x;
+        this.yLocation = y;
```

Comment:
Suggestion: When working with grids, I often see a `Coord` or `Position` type defined that might make some of the code in this class a bit more readable.  (For example, instead of referencing `xLocation` and `yLocation`, you would reference `position.X`.)